params/defaults.go

```diff
@@ -6,6 +6,9 @@ const (
 	// StatusDatabase path relative to DataDir.
 	StatusDatabase = "status-db"
 
+	ArchivesRelativePath        = "data/archivedata"
```

Comment:
I moved these constants to the "params" package, otherwise there will be a circular dependency between api and params